### PR TITLE
Add warning message in cli logs when dcv port is open to 0.0.0.0/0

### PR DESCRIPTION
**Pull request by demartinofra**
Add warning message in cli logs when dcv port is open to 0.0.0.0/0

Codecov: https://codecov.io/gh/aws/aws-parallelcluster

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/pcluster/config/validators.py [hunk 1]
```diff
@@ -28,6 +29,15 @@
     get_supported_os,
 )
 
+LOGFILE_LOGGER = logging.getLogger("cli_log_file")
+
+DCV_MESSAGES = {
+    "warnings": {
+        "access_from_world": "With this configuration you are opening dcv port ({port}) to the word (0.0.0.0/0). "
```
Comment (lukeseawalker) [inline cli/pcluster/config/validators.py]: word/world

Comment (demartinofra) [reply cli/pcluster/config/validators.py]: nice catch. thanks!

File: cli/pcluster/cli.py [hunk 2]
```diff
@@ -99,8 +100,9 @@ def config_logger():
 
     log_file_handler = RotatingFileHandler(logfile, maxBytes=5 * 1024 * 1024, backupCount=1)
     log_file_handler.setLevel(logging.DEBUG)
-    log_file_handler.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(name)s - %(message)s"))
+    log_file_handler.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(module)s - %(message)s"))
```
Comment (ddeidda) [inline cli/pcluster/cli.py]: Just out of curiosity, what is the goal of this change?

File: cli/pcluster/config/validators.py [hunk 3]
```diff
@@ -237,6 +247,13 @@ def dcv_enabled_validator(param_key, param_value, pcluster_config):
         if get_partition() not in get_supported_dcv_partition():
             errors.append("NICE DCV is not supported in the selected region '{0}'".format(get_region()))
 
+        if pcluster_config.get_section("dcv").get_param_value("access_from") == "0.0.0.0/0":
```
Comment (ddeidda) [inline cli/pcluster/config/validators.py]: Minor, could we use a meaningful utils constant name like CIDR_ALL_IPS?

File: cli/tests/requirements.txt [hunk 4]
```diff
@@ -5,4 +5,4 @@ pytest-datadir
 pytest-html
 pytest-mock
 assertpy
-discover # Needed for Python 2.6 compatibility
+codecov
```
Comment (ddeidda) [inline cli/tests/requirements.txt]: Should we maybe split this part about codecov on a separate commit?

File: cli/pcluster/cli.py (hunk 2 above)
Comment (demartinofra) [reply cli/pcluster/cli.py]: It's to always have the name of the module that originated the log record rather than the name of the logger that can be arbitrary

File: cli/pcluster/config/validators.py (hunk 3 above)
Comment (demartinofra) [reply cli/pcluster/config/validators.py]: done

File: cli/pcluster/constants.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (ddeidda) [inline cli/pcluster/constants.py]: Good idea, better than in the utils file.
